back-end/README.md

```diff
@@ -64,6 +77,21 @@ python -c "from django.core.management.utils import get_random_secret_key; print
     ```
     python manage.py runserver
     ```
+10. Start RabbitMQ server:
+- Open a new terminal tab and enter the following commands:
+    - **Linux** 
+    ```
+    sudo systemctl start rabbitmq-server
+    ```
+    - **Windows**
+    ```
+    rabbitmq-server start
+    ```
+11. Start the celery worker:
+ - Open a new terminal window and enter the following command:
+    ```
+    celery -A ComradeMarketPlace worker -l info

```

Comment:
The celery app that is set in `celery.py` is `cmp` . `ComradeMarketPlace ` does not exist

Reply:
resolved